### ISSUE TITLE
[BUG] Can't find tug and libgit2.so.1.3

### ISSUE THREAD

**Issue by superlevure** [b4nst/turbogit#112]
## Describe the bug

- `tug` is not available in Ubuntu default PATH
- Failed to install the correct `libgit2` lib

## To Reproduce

After installation on Ubuntu with: 
```
~ sudo apt install ./turbogit_v3.1.1_amd64_linux.deb
```
`tug` is not find in the PATH. Looking at the output of `dpkg -L turbogit`:
```
/.
/usr
/usr/share
/usr/share/doc
/usr/share/doc/turbogit
/usr/share/doc/turbogit/changelog.gz
/opt
/opt/local
/opt/local/bin
/opt/local/bin/tug
```
it seems that `/opt/local/bin` is not in Ubuntu default PATH. 

After having corrected that, launching `tug` gives: 
```
/opt/local/bin/tug: error while loading shared libraries: libgit2.so.1.3: cannot open shared object file: No such file or directory
```

Installing `libgit2` with:
```
sudo apt install libgit2-dev
```

Did not solve the issue unfortunately :/ 

## Versions

 - *tug*: 3.1.1

 - *OS*:  Ubuntu 18.04.6 LTS

**Comment by lucas-bremond** [comment]
I could indeed reproduce that issue, same here.

**Comment by lucas-bremond** [comment]
A couple of additional data points, on Ubuntu `20.04.3 LTS`, after installing `libgit2` using:

```shell
sudo apt install libgit2-dev
```

We end up with:

```shell
find /usr -name "*libgit2*"                                                                                                                                                                                               14s lucas@development-lucas
/usr/lib/x86_64-linux-gnu/pkgconfig/libgit2.pc
/usr/lib/x86_64-linux-gnu/libgit2.a
/usr/lib/x86_64-linux-gnu/libgit2.so
/usr/lib/x86_64-linux-gnu/libgit2.so.28
/usr/lib/x86_64-linux-gnu/libgit2.so.0.28.4
/usr/share/doc/libgit2-dev
/usr/share/doc/libgit2-28
```

Setting `export LD_LIBRARY_PATH=/usr/lib/x86_64-linux-gnu` does not work, as `libgit2.so.1.3` is still expected by `tug`.

**Comment by lucas-bremond** [comment]
What did the trick for me (temporary solution):

```shell
git clone https://github.com/libgit2/libgit2
cd libgit2
git checkout tags/v1.3.0
mkdir build && cd build
cmake .. -DCMAKE_INSTALL_PREFIX=/usr/local
sudo cmake --build . --target install
export LD_LIBRARY_PATH="/usr/local/lib"
```

**Comment by superlevure** [comment]
Yup, did the exact same thing as a workaround (but forgot to come here to comment of course)

**Comment by b4nst** [comment]
Indeed, libgit2 1.3. is not yet in Ubuntu 20.04.3 LTS default sources

**Comment by b4nst** [comment]
Should be fixed by version `4.0.0`. Libgit2 is now statically linked, so no more system dependencies.
